content/doc/book/system-administration/backing-up.adoc

```diff
@@ -97,7 +98,29 @@ Backup retention is based on file names that include both the job name and the b
 
 One of the most important things to remember when it comes to your Jenkins backup is to test your backup restore configuration on a Jenkins test server.
 
-== Using Jenkins backup plugins
+== How to back up Jenkins data
+
+=== File-based backup tools
+
+The tar command is widely used to create compressed archive files that can be moved easily from one disk to another disk, or machine to machine. Tar stands for tape archive, and is commonly referred to as tarball or tar, gzip and bzip in Linux. 
```

Comment:
Also has `--excludes` (or similar) that can be used to skip backing up `builds/` or `workspaces/` for a "configuration only" backup.

The robustness and flexibility here is really unmatched IMO.

Reply:
Can you give a few examples of a tar command for backing up the Jenkins Home directory, that does and does not use the `--excludes` setting?